app/uk/gov/hmrc/exports/scheduler/jobs/emails/SendEmailsJob.scala

```diff
@@ -97,20 +97,29 @@ class SendEmailsJob @Inject() (
 
       case Success(MissingData) =>
         logger.warn(s"Could not send email because some data is missing")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
 
       case Success(BadEmailRequest(msg)) =>
         logger.warn(s"Email service returned Bad Request response: [$msg]")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
 
       case Success(InternalEmailServiceError(msg)) =>
         logger.warn(s"Email service returned Internal Service Error response: [$msg]")
         logger.warn(s"Will try again in ${interval.toMinutes} minutes")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
     }
 
-  private def failSingle(workItem: WorkItem[SendEmailDetails]): Future[Unit] =
-    sendEmailWorkItemRepository.markAs(workItem.id, Failed).flatMap { _ =>
+  private def failOrCancelSingle(workItem: WorkItem[SendEmailDetails]): Future[Unit] = {
+    val status =
+      if (workItem.failureCount >= appConfig.parsingWorkItemsRetryLimit) {
```

Comment:
Won't this still reach 11 (if `parsingWorkItemsRetryLimit` is set to 10)?

Reply:
If a workItem attempts are equal to 10 times, it will be set to Cancelled, which will work for those created from now and in the future
For those already existing where the attempts are well over the limit already, they will also be Cancelled here

Tested for in `NotificationReceiptActionsRunnerSpec.scala`